src/views/studio/MyUploadsView/AssetsGroupUploadBar/AssetsGroupUploadBar.style.tsx

```diff
@@ -0,0 +1,116 @@
+import styled from '@emotion/styled'
+import { colors, sizes, zIndex, breakpoints, transitions } from '@/shared/theme'
+
+type ProgressbarProps = {
+  progress: number
+}
+
+type AssetsGroupUploadBarProps = {
+  isActive?: boolean
+}
+
+type DrawerProps = {
+  maxHeight?: number
+} & AssetsGroupUploadBarProps
+
+export const Container = styled.div`
+  position: relative;
+  margin-bottom: ${sizes(6)};
+`
+
+export const AssetsGroupUploadBarContainer = styled.div<AssetsGroupUploadBarProps>`
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding: ${sizes(4)};
+  width: 100%;
+  height: ${sizes(20)};
+  background-color: ${({ isActive }) => (isActive ? colors.gray[900] : colors.black)};
+  cursor: pointer;
+  transition: background-color ${transitions.timings.sharp} ${transitions.easing};
+  &:hover {
+    background-color: ${colors.gray[900]};
+  }
+`
+export const ProgressBar = styled.div<ProgressbarProps>`
+  --progress-bar-color: #b4bbff33;
```

Comment:
1. Why is this defined as a CSS variable? It don't think it's reused anywhere?
2. This is a color from our theme, let's use it via `colors`